package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "Together",
+  "name": "@together/together",
```

What do you think about:

```suggestion
  "name": "together",
```

Usually `@xxx/yyy` is used for published npm projects with [organisations scopes](https://docs.npmjs.com/about-organization-scopes-and-packages).